### DISCUSSION TITLE
Touch connection PIN X-X+Y-Y+

### DISCUSSION THREAD

**Discussion by boromyr** [Bodmer/TFT_eSPI#1043] [Q&A - Touch] [answered]
Hi, I wanted to know if it is possible to use the four pin X-X+Y-Y+ connection for touch controls, or if this library supports SPI mode for touch screen. The screen I need to use is not PCB mounted with XPT2046. 
Thank you.

**Comment by Bodmer** [comment, marked as answer]
[This library](https://github.com/s60sc/Adafruit_TouchScreen) has been used with an ESP32 and analogue touch screens but I do not provide support for it.

There may be other libraries for other processors but I have not tried them.

**Comment by boromyr** [comment]
Hi Bodmer, yesterday I managed to adapt your library with the one for analogue touch with my screen, and it works perfectly. 
Thank you very much!

**Reply by Bodmer** [reply]
Great, thanks for letting me know.
